docs/providers/aws/events/streams.md

```diff
@@ -329,3 +329,22 @@ functions:
           arn: arn:aws:dynamodb:region:XXXXXX:table/foo/stream/1970-01-01T00:00:00.000
           tumblingWindowInSeconds: 30
 ```
+
+## Setting FilterCriteria
+
+This configuration allows customers to filter event before lambda invocation. It accepts up to 5 filter criterion. If one event matches at least 1 pattern, lambda will process it.
+
+For more information and examples, read the [AWS event filtering documentation](https://docs.aws.amazon.com/lambda/latest/dg/invocation-eventfiltering.html)
```

Comment:
Let's maybe say `For more details and examples of filter patterns, please see...`